### PR TITLE
Trivial fixes to dart:ui docs.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: lib/ui/window.dart [hunk 1]
```diff
@@ -110,6 +110,16 @@ class Window {
...
   /// the Nexus 6 has a device pixel ratio of 3.5.
+  ///
+  /// Device pixels are also referred to as physical pixels. Logical pixels are
+  /// also referred to as device-independent or resolution-independent pixels.
+  ///
+  /// By definition, there are roughly 38 logical pixels per centimeter, or
+  /// about 96 logical pixels per inch, of the physical display. The value
+  /// returned by [devicePixelRatio] is ultimately obtained either from the
+  /// hardware itself, the device drivers, or a hard-coded value stored in the
+  /// operating system or firmware, and may vary somewhat wildly from that
```
Comment (jason-simmons) [inline lib/ui/window.dart]: what does "somewhat wildly" mean?
perhaps just say that it may be inaccurate?

Comment (Hixie) [inline lib/ui/window.dart]: ok, changed to "and may be inaccurate, sometimes by a significant margin".

File: lib/ui/window.dart [hunk 2]
```diff
@@ -127,7 +137,8 @@ class Window {
   WindowPadding _padding = WindowPadding.zero;
 
   /// A callback that is invoked whenever the [devicePixelRatio],
-  /// [physicalSize], or [padding] values change.
+  /// [physicalSize], or [padding] values change, for example when the device is
+  /// rotated.
```
Comment (abarth) [inline lib/ui/window.dart]: Or the application is resized (which can happen on Android now)

**Comment by abarth** [comment]
LGTM

**Pull request merged**